app/views/docs/permissions.phtml

```diff
@@ -23,18 +23,17 @@
 <div class="notice calm margin-top-large margin-bottom-large">
     <h4>Note</h4>
     
-    <p>You will need an active session to handle database transactions for your collections.</p>
-    <p>If you have rules where you allow for wild cards or role:guest in a collection, you can create an Anonymous Session before you make the API call.</p>
+    <p>You will need an active session to perform create, update, and delete type actions in your collections. If you have rules where you grant create, update, or delete access to all users or guests in a collection, create an Anonymous Session before you make the API call.</p>
```

Comment:
> If you have rules where you grant ...

Doesn't really sound right. Can it be rewritten ?

Reply:
Attempted, lmk if it's resolved.